cogeo_mosaic/backends/dynamodb.py

```diff
@@ -244,3 +271,11 @@ def _table_exist(self) -> bool:
             return True
         except self.table.meta.client.exceptions.ResourceNotFoundException:
             return False
+
+    def _mosaic_exist(self) -> bool:
+        """Check if the mosaic already Exist in the Table."""
+        pass
```

Comment:
how to?

Reply:
There is not easy way to check if a partition key/hash key is in a table. One if all mosaic's are guarenteed to have a metadata object stored under `quadKey = "-1"` I would recommend doing a `get_item({mosaicId = "...", "quadKey" = "-1"}` otherwise `putItem` and `updateItem` operations can be restricted to execute only if they meet a certain [condition](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.ConditionExpressions.html).

Reply:
> I would recommend doing a get_item({mosaicId = "...", "quadKey" = "-1"}
👍